lib/lumberjack/client.rb

```diff
@@ -69,10 +67,9 @@ def initialize(opts={})
         :port => 0,
         :address => "127.0.0.1",
         :ssl_certificate => nil,
-        :window_size => 5000
       }.merge(opts)
       @host = @opts[:address]
-      @window_size = @opts[:window_size]
+      @window_size = 1
```

Comment:
Rather, I'd like the window size to be > 1 by default and optionally controllable. Compression works much better over 100 events than it does with 1, and the latency per round-trip is amortized across the whole window size instead of per event.